curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646afc523275b302f16f948e.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 646afc523275b302f16f948e
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Now, you can see some gaps all around due to the default margin and padding of the project. you can fix that by targeting the universal selector `*` and setting the `margin` and `padding` to `0`. Then set the `box-sizing` property to `border-box` to make sure the padding and border are included in the total width and height of the element.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed